client/packages/system/src/Name/Details/Details.tsx

```diff
@@ -186,6 +194,21 @@ export const Details = ({ nameId, type = 'customer' }: DetailsProps) => {
             }
             labelWidthPercentage={19}
           />
+          {type === 'customer' && (
+            <DetailInputWithLabelRow
+              label={t('label.supply-level')}
+              labelWidthPercentage={19}
+              inputProps={{
+                disabled,
+                value:
```

Comment:
Have a look at dev tools inspector (when no properties defined):

"value" prop on "input" should not be null